crypto/evp/evp_enc.c

```diff
@@ -778,8 +776,8 @@ int EVP_DecryptUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl,
         return 0;
```

Comment:
Similarly to above. Or keep blocksize as int?

Reply:
Will do. I only broke this out to ease line length issues.